docs/space/import/README.md

```diff
@@ -10,23 +10,27 @@ Usage: contentful space import --content-file <file>
 Options:
   -h, --help                 Show help                                 [boolean]
   --space-id                 ID of the destination space                [string]
-  --environment-id           ID the environment in the destination space
-                                                    [string] [default: "master"]
-  --management-token         Contentful management API token            [string]
-  --content-file             JSON file that contains data to be import to your
-                             space                           [string] [required]
+  --environment-id           ID the environment in the destination space[string]
```

Comment:
did we remove the default value here?

Reply:
I think this is a leftover from the transition from the import cli to the generic cli where the default never was added (see https://github.com/contentful/contentful-import/blob/master/lib/usageParams.js#L12 and https://github.com/contentful/contentful-cli/blob/master/lib/cmds/space_cmds/import.js. But good point, the environment id option is out of sync between contentful-import and contentful-cli which is really a bit of a pain :) Not sure if the default from contentful-import gets picked up 🤔. I'll test it and try to align.

Reply:
Looks to me as if even though we don't explicitly set the default here anymore, we still default to `master` when setting the context here: https://github.com/contentful/contentful-cli/blob/master/lib/utils/middlewares.js#L42

We "removed" the default value when we moved everthing from the import-cli to the contentful-cli, but in practice, we'll still default to master if I see this correctly. In order to stay consistent, I added the default again to the cli option.

Reply:
Had to revert this change, as this would actually have introduced a breaking change I overlooked, as now we always would have overwritten `activeEnvironmentId` from the `.contentfulrc` config with `master`. So actually `master` has never been the default for the import command since it moved to contentful-cli and `.contentfulrc` was introduced, so the updated docs are correct in not saying so.

Reply:
does it mean if you don't provide a `.contenfulrc` file with an env id, the tool fails to execute? 
WHich makes sense to me, as you're not accidentally importing into your production env 🤔